lbz/events/api.py

```diff
@@ -65,13 +82,19 @@ def get_all_failed_events(self) -> List[BaseEvent]:
         return self._failed_events
 
     def send(self) -> None:
+        self._sent_events = []
+        self._failed_events = []
+
         try:
-            if self._pending_events:
-                entries = [self._create_eb_entry(event) for event in self._pending_events]
+            while self._pending_events:
+                events = self._pending_events[:MAX_EVENTS_TO_SEND_AT_ONCE]
```

Comment:
_FYI:_ Now, everything is in one method which looks self-descriptive to me, but if you think it is too long or something, I can move this part to a separate one 😉